web-console/4.0-designs/storage/object-storage-dashboard/object-storage-dashboard.md

```diff
@@ -0,0 +1,91 @@
+# Object Storage Dashboard￼
+
+## Vision
+Our main goal was to have an object's storage dashboard supported by MCG (NooBaa) that provide the user with at-a-glance information used for analysis and decision making. 
+
+## Object Dashboards
+
+![Object Storage Dashboard](img/object-storage-dashboard.png)
+
+**The objects dashboard includes the following cards:**
+- Details
+- Buckets (Object buckets and Object bucket claims)
+- Resources providers
+- Health
+- Data consumption 
+- Data resiliency 
+- Usage breakdown
+- Data Reduction 
+
+## Details Card
+![Details Crad](img/details-card.png)
+
+**The details card shows:**
+- connected service name and offer external link to the NooBaa (MCG) system. 
+- Provider name
+- MCG installed version
+
+## Buckets Card 
+![Buckets Crad](img/buckets-card.png)
+
+MCG - MCG/NooBaa Buckets are data containers that behave similarly to AWS S3 buckets. Users can set policies that will determine which resources will be user (per bucket) to store data.
+We will show a breakdown between static buckets and buckets claim. Each bucket type will have a status indication when something is wrong with X buckets (shows the number of buckets that are currently in warning/ error status). The number link should lead to the problematic buckets list. 
+
+## Resources Providers Card
+![Resources Providers](img/resource-card.png)
```

Comment:
@shirimordechay @yuvalgalanti 

We need to change the "Ceph" mentions in this card and others to "OCS."

Reply:
@julienlim isn't writing OCS creates a recursive loop? What happens in 4.3 when the ceph cluster is external?

Reply:
@yuvalgalanti Why changing Cepg to OCS? it doesn't make sense. Both Ceph and MCG are a part of OCS.

Reply:
+1 I think it can be confusing to call it OCS because of OCS in my eyes is the entire storage "world" when its installed and MCG and Ceph are part of that

Reply:
The reason I am suggesting to say OCS vs. Ceph is that it's going to be backed by OCS, and a lot of the references are OCS in other places, and it may get confusing to the user which one it is.  Also, in the future when we add RHCS support (external cluster), that's what most folks will think of as Ceph and not OCS.

@shirimordechay @yuvalgalanti @nimrod-becker